### PR TITLE
Verbose assertions

### PR DESCRIPTION
**Pull request by Yurasb**

### CODE REVIEW

File: tests/test_delete_all_links.py [hunk 1]
```diff
@@ -1,16 +1,25 @@
 import requests
+import allure
```
Comment (v1-wizard) [inline tests/test_delete_all_links.py]: Imports should sorted by alphabet.

File: tests/test_delete_all_links.py [hunk 2]
```diff
@@ -1,16 +1,25 @@
...
 from cerberus import Validator
 from lxml import html
 
 
+@allure.feature('Delete service')
```
Comment (v1-wizard) [inline tests/test_delete_all_links.py]: Delete all links handler?

Comment (v1-wizard) [reply tests/test_delete_all_links.py]: Why are you called all this handlers the 'service'?

**Review by v1-wizard — APPROVED** [review]

File: tests/test_delete_all_links.py (hunk 1 above)
Comment (Yurasb) [reply tests/test_delete_all_links.py]: Fixed in 034a15a5623401f2633bbd56277fe0c11ba2327c

File: tests/test_delete_all_links.py (hunk 2 above)
Comment (Yurasb) [reply tests/test_delete_all_links.py]: Renamed in ac0daf91b7b49eea8c0a4e2e2a411a3237508000

**Pull request merged**